includes/functions-html.php

```diff
@@ -828,7 +828,7 @@ function yourls_html_menu() {
 	/*
 	To display a notice:
 	$message = "<div>OMG, dude, I mean!</div>" );
-	yourls_add_action( 'admin_notices', create_function( '', "echo '$message';" ) );
+	yourls_add_action( 'admin_notices', function() use ( $message ) { echo "$message";} );
```

Comment:
```suggestion
	yourls_add_action( 'admin_notices', function() use ( $message ) { echo (string) $message; } );
```

Reply:
Thanks for taking the time to review this. I've applied the suggested changes.